### PR TITLE
document postcss usage more accurately

### PR DESCRIPTION
**Pull request by kylemac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,13 @@ This is nothing more than a light wrapper around a postcss configuration object.
...
 const cssStandards = require('spike-css-standards')
 
-postcss(cssStandards(/* options */))
+const standards = cssStandards(/* options */)
+// returns { parser: 'xxx', plugins: ['xxx'] }
+
+postcss(standards.plugins)
   .process(someCss)
   .then((res) => { console.log(res.content) })
+
```
Comment (jescalan) [inline README.md]: can we pass the parser here as well?

Comment (kylemac) [inline README.md]: i'm not sure i understand the question

Comment (jescalan) [inline README.md]: this example shows the plugins being passed, but not the parser

**Comment by jescalan** [comment]
💯 thank you!

**Pull request merged**